src/datastore/database.cpp

```diff
@@ -0,0 +1,168 @@
+#include "database.hpp"
+#include "src/utils/parser.hpp"
+
+/* Instantiate static map containers of teams and stadiums */
+std::map<int,Team> Database::teams;
+std::map<int,Stadium> Database::stadiums;
+
+/* Instantiate a static database object */
+Database* Database::m_database = new Database();
+
+/* Constructor */
+Database::Database()
+{}
+
+/**
+ * @brief Loads team, stadium, and souvenir data
+ *
+ * This functions takes a path, extracts all the data and instantiates
+ * team, stadium, and souvenir classes.
+ *
+ * @param filepath path to .csv data for teams,stadiums,and souvenirs
+ */
+void Database::loadFromFile(const std::string& filepath)
+{
+    // Used to traverser through souvenirs
+    unsigned int j;
```

Comment:
Declaration is too far from usage

Reply:
ROGER MOVE THIS ALREADY PLEASE. Why is this all the way up here?
![image](https://user-images.githubusercontent.com/5513122/56059195-695fe000-5d18-11e9-9c25-e005f732e9f6.png)